observability-test/database.ts

```diff
@@ -375,6 +375,115 @@ describe('Database', () => {
     });
   });
 
+  describe('batchCreateSessions', () => {
+    it('without error', done => {
+      const ARGS = [null, [{}]];
+      database.request = (config, callback) => {
+        callback(...ARGS);
+      };
+
+      database.batchCreateSessions(10, (err, sessions) => {
```

Comment:
let's not pass the session count directly, better if we make a variable first and then pass that variable to batchCreateSessions

Reply:
Ditto about that being pedantic yet this is a single usage.